docs/managers.rst

```diff
@@ -28,6 +28,43 @@ so you don't have to.
     >>> conn.num_requests
     3
 
+A :class:`.PoolManager` will create a new :doc:`ConnectionPool <pools>`
```

Comment:
Hm, I see it outside this diff too. Must be sphinx magic that I haven't seen before.